crates/primitives/src/chain_evm.rs

```diff
@@ -81,7 +83,34 @@ impl EVMChain {
             | Self::Abstract
             | Self::Berachain
             | Self::Hyperliquid => false,
-            Self::Optimism | Self::Base | Self::OpBNB | Self::World | Self::Ink | Self::Unichain => true,
+        }
+    }
+
+    // https://docs.zksync.io/zk-stack/running/quickstart
+    pub fn is_zkstack(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_zkstack(&self) -> bool {
```

maybe create enum `ChainStack: .native, .op, .zk` ? 

and then `var is_zkstack: chain_stack == .zk `